### PR TITLE
docs(examples): Add basic and with-prisma

### PR DESCRIPTION
**Pull request by Fumler**
I'm a fan of how Next.js has examples in the repo as this really helps convey how to do different things. The `with-prisma` example won't work unless #302 is merged though.

This is only a suggestion, feel free to close if not wanted.

### CODE REVIEW

**Comment by iaincollins** [comment]
Thanks @Fumler!

I'm a big fan of the `examples` dir too, although there are actually some caveats when including a lib (which I ran into with v1) so it's maybe worth a discussion - would like to describe them and see what people think. Will get back to this in a bit :-)

I definitely would like to have something like this, so if it doesn't go here, we should have it somewhere! Also I really want to learn more about the Prisma migration stuff! :-)

**Comment by yareyaredesuyo** [comment]
Prisma adapter feature looks like merged ( https://github.com/iaincollins/next-auth/pull/302 ), how about this PR ?

**Comment by iaincollins** [comment]
> Prisma adapter feature looks like merged ( #302 ), how about this PR ?

Things are too busy getting v3 ready to ship to handle issue of examples as well but we will get to it!

**Review by kadirselcuk — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -19,10 +19,11 @@ dist
 .env.development.local
 .env.test.local
 .env.production.local
+*.db
```
Comment (kadirselcuk) [inline .gitignore:22]: I'll leave PR here

**Review by kadirselcuk — APPROVED** [review]

**Pull request closed without merging**